lib/babe/verify_integration_test.go

```diff
@@ -49,7 +49,7 @@ func newTestVerificationManager(t *testing.T, genCfg *types.BabeConfiguration) *
 	require.NoError(t, err)
 
 	if genCfg == nil {
-		genCfg = genesisBABEConfig
+		genCfg = westendBABEConfig
```

Comment:
My goal in this whole cleanup (not this PR though) is to remove this helper function and just inject the cfg into createTestService helper